lib/indices.js

```diff
@@ -19,26 +19,20 @@ module.exports = [
     },
     {
         spec: { id: 1 },
-        options: {}
-    },
-    {
-        spec: {'amountAwarded': 1},
-        options: {}
-    },
-    {
-        spec: {'awardDate': 1},
-        options: {}
-    },
-    {
-        spec: {'grantProgramme.title': 1},
-        options: {}
+        options: {
+            name: 'ID'
+        }
     },
     {
-        spec: {'recipientOrganization.organisationType': 1},
-        options: {}
+        spec: { awardDate: -1 },
+        options: {
+            name: 'SortAwardDate'
+        }
     },
     {
-        spec: {'beneficiaryLocation.geoCode': 1},
-        options: {}
+        spec: { amountAwarded: 1 },
+        options: {
+            name: 'SortAmountAwarded'
+        }
```

Comment:
FYI – these changes won't take effect on production anymore so you'll need to either manually add them to the database or point your local environment at it and run the `./scripts/create-indices` command.

Reply:
I'd gathered. I don't think anything needs to be rerun at the moment (I'm not meaningfully changing anything, just removing unused and naming existing) so this can wait to be rerun